es_ES.json

```diff
@@ -0,0 +1,798 @@
+    "PLAYER_MAGNET": "Imán para jugadores",
+    "ENABLED": "Activado",
+    "CHAT": "Chat",
+    "AUTO_KICK_CHAT_SPAMMERS": "Auto-expulsar a los Spamers de Chat",
+    "DISABLE_FILTER": "Desactivar filtro",
+    "LOG_CHAT_MSG": "Registrar mensajes de chat",
+    "LOG_TXT_MSG": "Registrar mensajes de texto",
+    "IS_TEAM": "Es Equipo",
+    "SEND": "Enviar",
+    "CHAT_COMMANDS": "Comandos del chat",
+    "DEFAULT_CMD_PERMISSIONS": "Permisos de comandos por defecto",
+    "DECLOAK": "Decloak",
+    "REVEAL_OTR_PLAYERS": "Revelar jugadores OTR",
+    "FORCE_HOST": "Anfitrión forzado",
+    "FORCE_SESSION_HOST": "Forzar anfitrión de sesión",
+    "FORCE_SESSION_HOST_DESC": "Únete a otra sesión para aplicar los cambios. El anfitrión original de la sesión debe abandonarla o ser expulsado. Esta característica es fácilmente detectable por otros menús mod, utilizar con precaución",
+    "KICK_HOST_ON_JOIN": "Anfitrión de la patada durante la unión",
+    "FORCE_SCRIPT_HOST": "Forzar Script Host",
+    "REMOTE_NAME_SPOOFING": "Suplantación de nombres remotos",
+    "SPOOF_PLAYER_NAMES": "Falsear los nombres de otros jugadores",
+    "SPOOF_PLAYER_NAMES_DESC": "Requiere host de sesión. Los nombres suplantados no serán visibles localmente ni para el jugador cuyo nombre ha sido suplantado. Requiere que los jugadores se unan después de convertirse en anfitrión",
+    "ADVERTISE_YIMMENU": "Anunciar YimMenu",
+    "ADVERTISE_YIMMENU_DESC": "Anuncia YimMenu falsificando los nombres de los jugadores con variantes de 'YimMenu' de distintos colores. No podrás personalizar el nombre con esta opción activada",
+    "PLAYER_SPOOFED_NAME": "Nombre: ",
+    "ALL_PLAYERS": "Todos los jugadores",
+    "EXPLOSION_KARMA": "Explosión Karma",
+    "DAMAGE_KARMA": "Karma de daños",
+    "FORCE": "Fuerza",
+    "KILL_ALL": "Matar a todos",
+    "EXPLODE_ALL": "Explotar todo el mundo",
+    "INCLUDING_YOU": "Incluyéndote a ti",
+    "RAGDOLL_PLAYERS": "Jugadores Ragdoll",
+    "KICK_ALL_FROM_INTERIORS": "Patear a todos los de Interiores",
+    "TP_ALL_TO_APARTMENT": "TP Todo Al Apartamento",
+    "TP_ALL_TO_WAREHOUSE": "TP Todo al almacén",
+    "TP_ALL_TO_DARTS": "TP Todo a los Dardos",
+    "TP_ALL_TO_FLIGHT_SCHOOL": "TP Todos a la escuela de vuelo",
+    "TP_ALL_TO_MAP_CENTER": "TP Todo al centro de mapas",
+    "TP_ALL_TO_SKYDIVE": "TP Todos a saltar en paracaídas",
+    "TP_ALL_TO_MOC": "TP Todos a MOC",
+    "TP_ALL_TO_CASINO": "TP Todo al Casino",
+    "TP_ALL_TO_PENTHOUSE": "TP Todo Al Ático",
+    "TP_ALL_TO_ARCADE": "TP Todo a Arcade",
+    "TP_ALL_TO_MUSIC_LOCKER": "Teletransportar todos al armario de musica",
+    "TP_ALL_TO_RECORD_A_STUDIOS": "TP Todo Para Grabar Un Estudio",
+    "TP_ALL_TO_CUSTOM_AUTO_SHOP": "Teletransportar a todos a la tienda custom de coches",
+    "TP_ALL_TO_AGENCY": "TP Todo a la Agencia",
+    "TP_ALL_TO_FREAKSHOP": "Teletransportar a todos a Freakshop",
+    "TP_ALL_TO_MULTI_FLOOR_GARAGE": "TP Todo A Garaje De Varias Plantas",
```

Comment:
I add it here because I saw there are multiple translation using the same incorrect scheme. Correct it with 'Teletransportar a todos a... '.